sql/core/src/test/scala/org/apache/spark/sql/streaming/StreamingJoinSuite.scala

```diff
@@ -59,7 +59,7 @@ abstract class StreamingJoinSuite
     (input, df)
   }
 
-  protected def setupWindowedJoin(joinType: String)
+  protected def setupJoin(joinType: String)
```

Comment:
Probably I wasn't clear. This is a windowed join (though I also don't know whether this is an official representation) because we join two stream via "time window". Some tests don't join with time window and still have `windowed` in their name - that's what I meant.

Reply:
Sorry for missing these two at the first place. Updated.